subprojects/detekt.yml

```diff
@@ -253,8 +253,9 @@ formatting:
         autoCorrect: false
     Filename:
         active: false
+    # https://detekt.github.io/detekt/formatting.html#finalnewline
     FinalNewline:
-        active: false
+        active: true
         autoCorrect: false
         insertFinalNewLine: false
```

Comment:
In internal repository we use `insert_final_newline=true` in editorconfig.
I'd like to have a consistent code style whenever possible.

Reply:
we have `insert_final_newline = true` in this repo's editorconfig, yes, 
and this rule just checks for it

Reply:
I see. This is for autocorrection